docs/marionette.region.md

```diff
@@ -314,6 +316,60 @@ mainRegion.hasView() // true
 If you show a view in a region with an existing view, Marionette will
 [remove the existing View](#emptying-a-region) before showing the new one.
 
+### Non-Marionette Views
+
+Marionette Regions aren't just for showing Marionette Views - they can also
+display instances of regular [`Backbone.View`](http://backbonejs.org/#View).
+To do this, ensure your view defines a `render()` method and just treat it like
+a regular Marionette View:
+
+```javascript
+var Bb = require('backbone');
+var Mn = require('backbone.marionette');
+var _ = require('underscore');
+
+var MyChildView = Bb.View.extend({
+  render: function() {
+    this.$el.append('<p>Some text</p>');
+  }
+});
+
+var MyParentView = Mn.View.extend({
+  regions: {
+    child: '.child-view'
+  },
+
+  template: _.template('<div class="child-view"></div>'),
+
+  onRender: function() {
+    this.showChildView('child', new MyChildView());
+  }
+});
+```
+
+#### Showing Backgrid Views
```

Comment:
I dunno that this issue is specifically related to backgrid.

I don't think there's an issue for Marionette to fix here as things preference a Marionette.View or a typical flow for a Backbone.View

However any view that puts more than a wrapper `el` into the view's `el` and then attempts to render _inside_ that `el` will see this same issue.
ie:
```js
var View = Mn.View.extend({
  el: $('<div><p>')
});

var view = new View();
view.isRendered(); // true
```

The reason this seems _ok_ is that the typical case for a view is that view creates the `el` as a single element for rendering within _or_ the view is pre-rendered.  So if the view is _partially_ pre-rendered then the user will have to know to `render` it again before or after showing it in the region.  And that'll be true for backgrid or any other case where this is a thing.